apps/src/sites/studio/pages/congrats/index.js

```diff
@@ -18,9 +18,11 @@ $(document).ready(function () {
   const userType = congratsData.current_user
     ? congratsData.current_user.user_type
     : 'signedOut';
-  const userName = congratsData.current_user
-    ? congratsData.current_user.name
-    : '';
+
+  const givenName = congratsData.current_user.properties?.given_name;
+  const familyName = congratsData.current_user.properties?.family_name;
+  const userName = givenName && familyName ? `${givenName} ${familyName}` : '';
```

Comment:
Should it default to `current_user.name` then default to `''` if none of those are present? I think we still want to keep the `name` as a backup if they don't have the given and family names.